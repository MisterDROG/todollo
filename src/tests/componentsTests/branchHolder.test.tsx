import BranchHolder from '../../components/branchHolder/branchHolder';
import { renderWithProviders } from '../testUtils/testFunctions';

describe('Branch holder tests', () => {
    test('Is branch holder renders right', () => {
        const { container } = renderWithProviders(<BranchHolder />)
        const divBranchHolder = container.querySelector('.branch-holder')

        expect(divBranchHolder).toBeInTheDocument()
        expect(container).toMatchSnapshot()
    })

})