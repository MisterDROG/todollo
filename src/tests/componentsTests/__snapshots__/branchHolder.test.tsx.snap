// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Branch holder tests Is branch holder renders right 1`] = `
<div>
  <div
    class="branch-holder"
  >
    <div
      class="branch"
    >
      <div
        class="branch-container"
      >
        <div
          class="branch__name-container"
        >
          <p
            class="branch__name"
          >
            Loading...
          </p>
          <button
            class="branch__button-delete"
          >
            X
          </button>
        </div>
        <div
          class="card"
          draggable="true"
          style="margin-top: 10px; margin-bottom: 10px;"
        >
          <div
            class="card__header"
          >
            <p
              class="card__text"
            >
              Loading...
            </p>
            <button
              class="card__button-delete"
            >
              X
            </button>
          </div>
          <div
            class="card__info"
          >
            <img
              alt="clockDate image"
              class="card__logo-img"
              src="clockDateImage.png"
            />
            <p
              class="card__date"
            >
              loading...
            </p>
          </div>
          <button
            class="card__button-status_undone"
          >
            unDone
          </button>
        </div>
        <div
          class="card"
          draggable="true"
          style="margin-top: 10px; margin-bottom: 10px;"
        >
          <div
            class="card__header"
          >
            <p
              class="card__text"
            >
              Loading...
            </p>
            <button
              class="card__button-delete"
            >
              X
            </button>
          </div>
          <div
            class="card__info"
          >
            <img
              alt="clockDate image"
              class="card__logo-img"
              src="clockDateImage.png"
            />
            <p
              class="card__date"
            >
              loading...
            </p>
          </div>
          <button
            class="card__button-status_undone"
          >
            unDone
          </button>
        </div>
        <div
          class="card"
          draggable="true"
          style="margin-top: 10px; margin-bottom: 10px;"
        >
          <div
            class="card__header"
          >
            <p
              class="card__text"
            >
              Loading...
            </p>
            <button
              class="card__button-delete"
            >
              X
            </button>
          </div>
          <div
            class="card__info"
          >
            <img
              alt="clockDate image"
              class="card__logo-img"
              src="clockDateImage.png"
            />
            <p
              class="card__date"
            >
              loading...
            </p>
          </div>
          <button
            class="card__button-status_undone"
          >
            unDone
          </button>
        </div>
        <form
          class="branch__form-new-card"
        >
          <input
            class="branch__input-new-card"
            placeholder="New Task..."
            type="text"
            value=""
          />
          <button
            class="branch__button-add-card"
          >
            Add
          </button>
        </form>
      </div>
    </div>
    <div
      class="branch"
    >
      <div
        class="branch-container"
      >
        <div
          class="branch__name-container"
        >
          <p
            class="branch__name"
          >
            Loading...
          </p>
          <button
            class="branch__button-delete"
          >
            X
          </button>
        </div>
        <div
          class="card"
          draggable="true"
          style="margin-top: 10px; margin-bottom: 10px;"
        >
          <div
            class="card__header"
          >
            <p
              class="card__text"
            >
              Loading...
            </p>
            <button
              class="card__button-delete"
            >
              X
            </button>
          </div>
          <div
            class="card__info"
          >
            <img
              alt="clockDate image"
              class="card__logo-img"
              src="clockDateImage.png"
            />
            <p
              class="card__date"
            >
              loading...
            </p>
          </div>
          <button
            class="card__button-status_undone"
          >
            unDone
          </button>
        </div>
        <div
          class="card"
          draggable="true"
          style="margin-top: 10px; margin-bottom: 10px;"
        >
          <div
            class="card__header"
          >
            <p
              class="card__text"
            >
              Loading...
            </p>
            <button
              class="card__button-delete"
            >
              X
            </button>
          </div>
          <div
            class="card__info"
          >
            <img
              alt="clockDate image"
              class="card__logo-img"
              src="clockDateImage.png"
            />
            <p
              class="card__date"
            >
              loading...
            </p>
          </div>
          <button
            class="card__button-status_undone"
          >
            unDone
          </button>
        </div>
        <div
          class="card"
          draggable="true"
          style="margin-top: 10px; margin-bottom: 10px;"
        >
          <div
            class="card__header"
          >
            <p
              class="card__text"
            >
              Loading...
            </p>
            <button
              class="card__button-delete"
            >
              X
            </button>
          </div>
          <div
            class="card__info"
          >
            <img
              alt="clockDate image"
              class="card__logo-img"
              src="clockDateImage.png"
            />
            <p
              class="card__date"
            >
              loading...
            </p>
          </div>
          <button
            class="card__button-status_undone"
          >
            unDone
          </button>
        </div>
        <form
          class="branch__form-new-card"
        >
          <input
            class="branch__input-new-card"
            placeholder="New Task..."
            type="text"
            value=""
          />
          <button
            class="branch__button-add-card"
          >
            Add
          </button>
        </form>
      </div>
    </div>
    <div
      class="branch"
    >
      <div
        class="branch-container"
      >
        <div
          class="branch__name-container"
        >
          <p
            class="branch__name"
          >
            Loading...
          </p>
          <button
            class="branch__button-delete"
          >
            X
          </button>
        </div>
        <div
          class="card"
          draggable="true"
          style="margin-top: 10px; margin-bottom: 10px;"
        >
          <div
            class="card__header"
          >
            <p
              class="card__text"
            >
              Loading...
            </p>
            <button
              class="card__button-delete"
            >
              X
            </button>
          </div>
          <div
            class="card__info"
          >
            <img
              alt="clockDate image"
              class="card__logo-img"
              src="clockDateImage.png"
            />
            <p
              class="card__date"
            >
              loading...
            </p>
          </div>
          <button
            class="card__button-status_undone"
          >
            unDone
          </button>
        </div>
        <div
          class="card"
          draggable="true"
          style="margin-top: 10px; margin-bottom: 10px;"
        >
          <div
            class="card__header"
          >
            <p
              class="card__text"
            >
              Loading...
            </p>
            <button
              class="card__button-delete"
            >
              X
            </button>
          </div>
          <div
            class="card__info"
          >
            <img
              alt="clockDate image"
              class="card__logo-img"
              src="clockDateImage.png"
            />
            <p
              class="card__date"
            >
              loading...
            </p>
          </div>
          <button
            class="card__button-status_undone"
          >
            unDone
          </button>
        </div>
        <div
          class="card"
          draggable="true"
          style="margin-top: 10px; margin-bottom: 10px;"
        >
          <div
            class="card__header"
          >
            <p
              class="card__text"
            >
              Loading...
            </p>
            <button
              class="card__button-delete"
            >
              X
            </button>
          </div>
          <div
            class="card__info"
          >
            <img
              alt="clockDate image"
              class="card__logo-img"
              src="clockDateImage.png"
            />
            <p
              class="card__date"
            >
              loading...
            </p>
          </div>
          <button
            class="card__button-status_undone"
          >
            unDone
          </button>
        </div>
        <form
          class="branch__form-new-card"
        >
          <input
            class="branch__input-new-card"
            placeholder="New Task..."
            type="text"
            value=""
          />
          <button
            class="branch__button-add-card"
          >
            Add
          </button>
        </form>
      </div>
    </div>
    <form
      class="branch-holder__form-new-branch"
    >
      <input
        class="branch-holder__input-new-branch"
        placeholder="New Branch..."
        type="text"
        value=""
      />
      <button
        class="branch-holder__button-add-branch"
      >
        Add
      </button>
    </form>
  </div>
</div>
`;
